.github/workflows/tiobe.yaml

```diff
@@ -5,14 +5,20 @@ on:
   schedule:
     - cron:  '0 7 1 * *'
 
+permissions: {}
+
 jobs:
   TICS:
     runs-on: [self-hosted, reactive, amd64, tiobe, noble]
     steps:
       - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
 
       - name: Set up Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24'
```

Comment:
Can we use `go-version-file` (to get the version from go.mod) instead? https://github.com/actions/setup-go/?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Reply:
TIL :)